microsite/data/plugins/cloudify.yaml

```diff
@@ -3,7 +3,7 @@ title: Cloudify
 author: Cloudify
 authorUrl: https://cloudify.co/
 category: Orchestration
-description: Load blueprints from desired Cloudify Manager instance
-documentation: https://github.com/Cloudify-PS/backstage-cloudify-plugin#readme
+description: Cloudify provides a remote execution and environment management backend that handles the provisioning and continuous update of Kubernetes, Terraform, Ansible, CloudFormation, Azure ARM, VRO based environments through a single API endpoint.
```

Comment:
This description is fairly long and might look a bit strange on the website. Could you perhaps shorten this a bit? 🙏